src/debug/debug.yaml

```diff
@@ -60,3 +60,14 @@
     name: reputations
     schema:
       $ref: '#/components/schemas/Reputations'
+- name: debug_bundler_addUserOps
+  summary: Adds the userOps array to the local mempool
+  params:
+    - name: userOperations
+      required: true
+      schema:
+        $ref: '#/components/schemas/UserOperation'
```

Comment:
something is missing here... it should be "array" (or defined to add a single userop)